ClientApp/viking-vault/src/ProfileData.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 import { render } from "react-dom";
 import './ProfileData.css';
 import { Redirect } from 'react-router-dom';
-import {constants} from "./ConstantVariables";
+import { constants } from "./ConstantVariables";
 
 const API_URL = constants.baseUrl+"userprofilepages";
```

Comment:
Small suggestion, you can use string interpolation here instead of string concatenation. The syntax is the following


\`${constants.baseUrl}userprofilepages\`;

Reply:
Is it more efficient or why?

Reply:
Not because of the efficiency as this operation is really small, but more from the perspective of code readability. A single concatenation is usually ok, but when you concatenate 3 or more items it becomes easier to read while using string interpolation, for this reason and to maintain consistency in the application, I think that concatenations like these should also be done using interpolation